torch/onnx/_internal/fx/function_dispatcher.py

```diff
@@ -180,13 +180,23 @@ def _create_op_overload_to_exporter_key_table() -> Dict[
                 table[op_overload] = op_overload_packet._qualified_op_name
     # TODO(justinchuby): is baddbmm different?
     table[torch.ops.aten.baddbmm.default] = "aten::baddbmm"
+    # TODO(titaiwang): aten::sym_size doesn't have overload,
+    # but works fine as overloadpacket
+    table[torch.ops.aten.sym_size] = "aten::sym_size"
     return table
 
 
 # Dictionary that maps torch.ops.aten.* to exporter look up key; e.g.,
 # _OP_OVERLOAD_TO_EXPORTER_KEY_TABLE[torch.add.Tensor] is "aten::add".
 _OP_OVERLOAD_TO_EXPORTER_KEY_TABLE = _create_op_overload_to_exporter_key_table()
 
+_BUILTIN_OVERLOAD_TO_EXPORTER_KEY_TABLE = {
+    "mul": "aten::mul",
```

Comment:
Don't do this. It will map all `+`, `-`, `*`, and `/` to `aten::*`. Let's do
```
_BUILTIN_OVERLOAD_TO_EXPORTER_KEY_TABLE = {
    "mul": "__builtin_mul__",
    "add": "__builtin_add__",
    "sub": "__builtin_sub__",
    "truediv": "__builtin_truediv__",
}
```

Then, create the corresponding functions in dispatch table such as
```
def __builtin_mul__(self, other):
    if x is not Tensor or y is not Tensor:
      raise
    ops.core.aten_add(x, y)
    
```

Basically, I want to check the types before calling those built's exporters.

Reply:
Per our discussion, we can replace this dictionary with
```
# This means all SymInt's are mapped to `torch.Tensor` before mapping ONNX Script.
torch.SymInt.__add__: "aten::add"
...
```

Reply:
I use operator.add instead. LMK if it's good.